sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 1.23.1 (Unreleased)
+## 1.24.0b1 (Unreleased)
 
 ### Features Added
 
+- Expanded the set of acceptable values for environment variable `AZURE_TOKEN_CREDENTIALS` to allow for selection of a specific credential in the `DefaultAzureCredential` chain. At runtime, only the specified credential will be used when acquiring tokens with `DefaultAzureCredential`. For example, setting `AZURE_TOKEN_CREDENTIALS=WorkloadIdentityCredential` will make `DefaultAzureCredential` use only `WorkloadIdentityCredential`.
```

Comment:
Maybe we need to update README to list all valid values for AZURE_TOKEN_CREDENTIALS?